sunshine/nvhttp.cpp

```diff
@@ -437,13 +437,21 @@ void serverinfo(std::shared_ptr<typename SimpleWeb::ServerBase<T>::Response> res
   tree.put("root.appversion", VERSION);
   tree.put("root.GfeVersion", GFE_VERSION);
   tree.put("root.uniqueid", config::nvhttp.unique_id);
-  tree.put("root.mac", "42:45:F0:65:D6:F4");
+  tree.put("root.mac", "00:00:00:00:00:00");
+  tree.put("root.MaxLumaPixelsHEVC", config::video.hevc_mode > 0 ? "1869449984" : "0");
   tree.put("root.LocalIP", local_ip);
 
-  if(config::nvhttp.external_ip.empty()) {
-    tree.put("root.ExternalIP", local_ip);
```

Comment:
`ExternalIP` is optional (and not provided in recent GFE versions), but specifying one will override Moonlight's external IP address detection using STUN when the PC is discovered over mDNS. Therefore, we should only provide one if the user has explicitly provided it.